roles/openshift_aws/defaults/main.yml

```diff
@@ -8,6 +8,8 @@ openshift_aws_create_scale_group: True
 openshift_aws_kubernetes_cluster_status: owned  # or shared
 openshift_aws_node_group_type: master
 
+
+openshift_aws_deployment_type: "{{ openshift_deployment_type | default('origin')}}"
```

Comment:
I would prefer we keep openshift_deployment_type and fail hard without it.

Reply:
Ok, I can remove.

Reply:
Ok, after further investigation here is the reasoning for keeping this variable.

`openshift_deployment_type` is defined in the inventory.  This runs on localhost which currently is not part of the inventory.  I'd prefer to default the behavior to what is here.  If `openshift_deployment_type` is defined then it will be used.  That way we obey the inventory if called on a host in the inventory and supply a default when not.